opentracing-spring-web/src/test/java/io/opentracing/contrib/spring/web/client/TracingRestTemplateInterceptorTest.java

```diff
@@ -99,7 +99,7 @@ public void testStandardTagsWithPort() {
         Assert.assertEquals("GET", mockSpan.tags().get(Tags.HTTP_METHOD.getKey()));
         Assert.assertEquals(url, mockSpan.tags().get(Tags.HTTP_URL.getKey()));
         Assert.assertEquals(200, mockSpan.tags().get(Tags.HTTP_STATUS.getKey()));
-        Assert.assertEquals((short)4000, mockSpan.tags().get(Tags.PEER_PORT.getKey()));
```

Comment:
It is, this bumps filter version which uses ot 0.21 where port has been changed to `int`.